projects/vdk-plugins/vdk-plugin-control-cli/src/vdk/plugin/control_cli_plugin/vdk_plugin_control_cli.py

```diff
@@ -26,7 +26,7 @@
 from vdk.plugin.control_cli_plugin import control_service_configuration
 
 
-@hookimpl
+@hookimpl(tryfirst=True)
```

Comment:
can you add a comment as to why it's set as 'tryfirst' . 

I advise on building a habit to add comments if you think the reason for the code is non-obvious.

Reply:
noted